src/lib/config.js

```diff
@@ -3,26 +3,41 @@ import enquirer from 'enquirer';
 import findPrefix from 'find-npm-prefix';
 import fs from 'fs-extra';
 import Client from 'mina-signer';
-import { PrivateKey, PublicKey } from 'o1js';
+import { Lightnet, Mina, PrivateKey, PublicKey } from 'o1js';
 import { getBorderCharacters, table } from 'table';
 import Constants from './constants.js';
-import step from './helpers.js';
+import step, { isMinaGraphQlEndpointAvailable } from './helpers.js';
 import prompts from './prompts.js';
 
 const log = console.log;
 
 /**
- * Show existing deploy aliases in `config.json` and allow a user to add a new
- * deploy alias and url--and generate a key pair for it.
+ * Show existing deploy aliases in `config.json` and allow user to add a new
+ * deploy alias.
+ * @param {object}  argv - The arguments object provided by yargs.
+ * @param {boolean} argv.list - Whether to list the available deploy aliases and their configurations.
```

Comment:
This is a good idea to allow the user to list their deploy aliases when they want! For my understanding what is the user flow to list the deploy aliases? Is it just zk `config --list`.

Reply:
Yes, it is simple `zk config --list`, then we have `zk config --lightnet` and then `zk config` which defaults to same interactive behaviour as before.